templates/fetch_git.nix.eex

```diff
@@ -1,4 +1,4 @@
     <%= name %> = fetchGit {
-      url = <%= url %>;
+      url = "<%= url %>";
```

Comment:
Either syntax is supported though, but quoting just in case is probably a good idea.